.github/workflows/tests.yaml

```diff
@@ -27,3 +28,22 @@ jobs:
       run: make
     - name: Run tests
       run: make check
+  build_tflite:
+    runs-on: ubuntu-18.04
+    container: ridgerun/r2inference:v0.1.3
+    env:
+      TENSORFLOW_PATH: /root/r2inference/backends/tflite/v2.0.1/include/tensorflow
+      CPPFLAGS: "-I${TENSORFLOW_PATH} -I${TENSORFLOW_PATH}/tensorflow/lite/tools/make/downloads/flatbuffers/include"
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install backend version
+      run: |
+        cp /root/r2inference/backends/tflite/v2.0.1/binaries/libtensorflow-lite.a /usr/local/lib/    
+    - name: Configure the project
+      run: |
+        export CPPFLAGS="${{ env.CPPFLAGS }}" 
```

Comment:
Why is this necessary?

Reply:
To my understanding, when a variable is defined globally it can be referenced anywhere within the workflow, but when it comes to steps with scripts or any child process that needs such variable it needs to be explicitly passed. The export wasn't necessary, so I  removed it.